src/math/vec.zig

```diff
@@ -374,9 +374,61 @@ pub fn Vec(comptime n_value: usize, comptime Scalar: type) type {
 
             return min_scalar;
         }
+
+        /// Checks for approximate (tolerence) equality between two vectors
+        /// of the same type and dimensions
+        pub inline fn equalsApprox(a: *const VecN, b: *const VecN, tolerance: T) bool {
```

Comment:
```suggestion
        pub inline fn eqlApprox(a: *const VecN, b: *const VecN, tolerance: T) bool {
```

I intend to standardize on `eql`, matching `std.mem.eql` and others in the stdlib.

Reply:
Makes absolute sense. Will adjust.